src/Compilers/VisualBasic/Test/Semantic/Semantics/BinaryOperators.vb

```diff
@@ -1211,6 +1211,10 @@ End Class
             Assert.Equal(MethodKind.BuiltinOperator, symbol1.MethodKind)
             Assert.True(symbol1.IsImplicitlyDeclared)
 
+            Dim synthesizedMethod = compilation.CreateBuiltinOperator(
+                symbol1.Name, symbol1.ReturnType, symbol1.Parameters(0).Type, symbol1.Parameters(1).Type, symbol1.IsCheckedBuiltin)
+            Assert.Equal(synthesizedMethod, symbol1)
```

Comment:
hooks to validate existing testcases where we create builtin-operators can properly be created with this new api.